packages/modules/create-itwin-viewer/bin.js

```diff
@@ -0,0 +1,184 @@
+import { bold, cyan, green } from "kleur/colors";
+import { exec } from "child_process";
+import prompts from "prompts";
+import path from "path";
+import fs from "fs-extra";
+import { uiConfigurations } from "./config/ui.mjs";
+import { packageJson } from "./config/package.mjs";
+import { appConfiguration } from "./config/app.mjs";
+import validatePackageName from "validate-npm-package-name";
+
+function finalize(appName) {
+  const endMsg = bold(green("Application generation complete!"));
+  const nextSteps = "Next Steps:";
+  let i = 1;
+  const chDir = bold(cyan(`${i++}. cd ${appName}`));
+  const startApp = bold(
+    cyan(`${i++}. npm start (or yarn start / pnpm start) to start debugging`)
+  );
+  const editApp = bold(
+    cyan(
+      `${i++}. open the directory in your editor of choice to edit the application`
+    )
+  );
+  console.log(endMsg);
+  console.log(nextSteps);
+  console.log(chDir);
+  console.log(editApp);
+  console.log(startApp);
+}
+
+function installDependencies(appRoot, appName) {
+  const npmMsg = "Installing dependencies...";
+  console.log(npmMsg);
+  process.chdir(appRoot);
+  exec("npm install", () => {
+    finalize(appName);
+  });
+}
+
+function copyTemplate(appRoot, platform) {
+  const generatorRoot = new URL(".", import.meta.url).pathname;
```

Comment:
I believe this line causes an issue on Windows (unless its just my configuration). 
E.g., at this point in my test run, the appRoot is `"C:\\Users\\John.DiMatteo\\repos\\test"` (OK).  
But generatorRoot is `"/C:/Users/John.DiMatteo/repos/viewer.cli/packages/modules/create-itwin-viewer/"` (not OK, beginning forward slash is invalid).
And then templateRoot becomes: `"C:\\C:\\Users\\John.DiMatteo\\repos\\viewer.cli\\packages\\modules\\create-itwin-viewer\\templates\\web"` (ERROR path not found).

Reply:
this should be resolved

Reply:
Just made [a commit](https://github.com/iTwin/viewer/pull/155/commits/a7b338c57ada8c76fe6ed24c93343178f21f4ae3) which should finally fix the issue - just double check I didn't break it on your end @kckst8 and you can close this